discussion/app/controllers/CommentsController.scala

```diff
@@ -144,11 +144,10 @@ class CommentsController(csrfConfig: CSRFConfig, val discussionApi: DiscussionAp
           RevalidatableResult.Ok(views.html.discussionComments.discussionPage(page))
         }
       }
+    } recover {
+      case NonFatal(e) =>
+        InternalServerError(s"Discussion $key cannot be retrieved, has error: ${e.getMessage}")
```

Comment:
I would be a little wary about letting an error message out, also it won't give all the info.  May be worth keeing the message the same but sending the exception to the logger, we can view the logs with kibana so it's quite easy.